weave/trace/weave_client.py

```diff
@@ -1151,9 +1152,8 @@ def send_start_call() -> bool:
 
         def on_complete(f: Future) -> None:
             try:
-                if f.result() and not call_context.get_current_call():
-                    if _should_print_call_link:
-                        print_call_link(call)
+                if _should_print_call_link and f.result() and not _current_call:
```

Comment:
for readability can you assign this to a var or something? I don't think its immediately obvious why we wouldn't want to print the call if "not current call".

Reply:
yep